contracts/package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "@chainlink/contracts-ccip",
-  "version": "1.5.0",
-  "description": "Chainlink CCIP smart contracts",
+  "name": "@chainlink/contracts",
+  "version": "1.2.0",
+  "description": "Chainlink smart contracts",
   "author": "Chainlink devs",
-  "license": "BUSL-1.1",
+  "license": "MIT",
```

Comment:
Is this ok? 🤔

Reply:
Oh this is interesting, not only the license but also the version and all other changes in that file